src/Raven.Server/Integrations/PostgreSQL/PgServer.cs

```diff
@@ -112,7 +112,7 @@ private void Stop()
             {
                 try
                 {
-                    task.Wait();
+                    task.Wait(TimeSpan.FromSeconds(3));
```

Comment:
What happens if this returns false?

Reply:
We're closing the server so the connection will be aborted

Reply:
ok